### PR TITLE
Challenge/anna sala cv full stack

### PR DESCRIPTION
**Pull request by annasm07**

### CODE REVIEW

File: anna-sala/challenges/cv-full-stack/frontend/src/redux/actions/actionCreators.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import axios from 'axios';
+import actionTypes from './actionTypes';
+
+const url = 'http://localhost:3535/curriculum';
```
Comment (SantiagoMC52) [inline anna-sala/challenges/cv-full-stack/frontend/src/redux/actions/actionCreators.js:4]: Podrías añadir esta url en un fichero .env! En el fichero .env añades: `REACT_APP_URL=<AQUÍ VA LA URL>` luego cambias la línea 4 por` const url = process.env.REACT_APP_URL;`

**Review by SantiagoMC52 — APPROVED** [review]

**Review by blopez-dev — APPROVED** [review]

File: anna-sala/challenges/cv-full-stack/backend/src/controllers/curriculumController.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+const debug = require('debug')('server:curriculumController');
+const CV = require('../model/cvModel');
+
+function curriculumController() {
+  debug('inside getCV');
+  async function getCV(req, res) {
+    const curriculum = await CV.find();
```
Comment (fredyzz) [inline anna-sala/challenges/cv-full-stack/backend/src/controllers/curriculumController.js:7]: Nice use of a try catch :P

File: anna-sala/challenges/cv-full-stack/backend/.eslintrc.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  env: {
+    commonjs: true,
+    es2021: true,
+    node: true,
```
Comment (davzlum) [inline anna-sala/challenges/cv-full-stack/backend/.eslintrc.js:5]: Te falta es "jest: true" para poder testear el backend

File: anna-sala/challenges/cv-full-stack/backend/src/controllers/curriculumController.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  }
+  async function createCV(req, res) {
+    const newCV = new CV(req.body);
+    debug(newCV);
+    try {
+      await newCV.save();
+      res.json(newCV);
+    } catch (error) {
+      debug(error);
+      res.send(error);
```
Comment (fredyzz) [inline anna-sala/challenges/cv-full-stack/backend/src/controllers/curriculumController.js:18]: It's a good practice to return status code

File: anna-sala/challenges/cv-full-stack/backend/src/model/cvModel.js [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+    awards: [{
+      timeFrame: String,
+      description: String,
+    }],
+    leadership: [{
+      timeFrame: String,
+      title: String,
+      place: String,
+      description: [String],
+    }],
```
Comment (fredyzz) [inline anna-sala/challenges/cv-full-stack/backend/src/model/cvModel.js:43]: I would have paid for you to give me this model

**Review by davzlum — APPROVED** [review]
Nice work teammate!

**Review by fredyzz — APPROVED** [review]
I see in you millionaire skills, nice work 👏

File: anna-sala/challenges/cv-full-stack/backend/src/model/cvModel.js (hunk 5 above)
Comment (annasm07) [reply anna-sala/challenges/cv-full-stack/backend/src/model/cvModel.js:43]: hahahah

**Pull request merged**